$NetBSD: patch-lib_tomoe_c,v 1.2 2014/05/05 06:26:53 obache Exp $

* It is no longer possible to include individual headers since glib2-2.32 

--- lib/tomoe.c~	2007-06-06 07:35:53.000000000 +0000
+++ lib/tomoe.c
@@ -26,7 +26,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <glib/garray.h>
+#include <glib.h>
 #include "tomoe.h"
 
 static gboolean initialized = FALSE;
